src/Symfony/Component/Notifier/Bridge/Bluesky/BlueskyOptions.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Symfony\Component\Notifier\Bridge\Bluesky;
+
+use Symfony\Component\Mime\Part\File;
```

Comment:
Do we already have a dependency in composer.json for `symfony/mime`?

Reply:
No.

`composer.json` have been updated.